host/4/bullseye/amd64/python/python311/python311-buildenv.Dockerfile

```diff
@@ -0,0 +1,47 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/mirror/docker/library/python:3.11-slim
+
+ENV LANG=C.UTF-8 \
+    ACCEPT_EULA=Y \
+    AzureWebJobsScriptRoot=/home/site/wwwroot \
+    HOME=/home \
+    FUNCTIONS_WORKER_RUNTIME=python \
+    ASPNETCORE_URLS=http://+:80 \
+    DOTNET_RUNNING_IN_CONTAINER=true \
+    DOTNET_USE_POLLING_FILE_WATCHER=true
+
+# Install Python dependencies
+RUN apt-get update && \
+    apt-get install -y wget apt-transport-https curl gnupg locales && \
+    echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
+    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
+    curl https://packages.microsoft.com/config/debian/11/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
+    # Needed for libss1.0.0 and in turn MS SQL
+    echo 'deb http://security.debian.org/debian-security stable-security main' >> /etc/apt/sources.list && \
+    # install necessary locales for MS SQL
+    echo 'en_US.UTF-8 UTF-8' > /etc/locale.gen && \
+    locale-gen && \
+    apt-get update && \
+    ACCEPT_EULA=Y apt-get install -y unixodbc msodbcsql17 mssql-tools &&\
+    apt-get install -y --no-install-recommends ca-certificates \
+    libc6 libgcc1 libgssapi-krb5-2 libicu67 libssl1.1 libstdc++6 zlib1g &&\
+    apt-get install -y libglib2.0-0 libsm6 libxext6 libxrender-dev xvfb binutils\
+    binutils libgomp1 liblttng-ust0 && \
+    rm -rf /var/lib/apt/lists/*
+    # MS SQL related packages: unixodbc msodbcsql17 mssql-tools
+    # .NET Core dependencies: --no-install-recommends ca-certificates libc6 libgcc1 libgssapi-krb5-2 libicu67 libssl1.1 libstdc++6 zlib1g
+    # OpenCV dependencies:libglib2.0-0 libsm6 libxext6 libxrender-dev xvfb
+    # binutils: binutils
+    # OpenMP dependencies: libgomp1
+    # Azure ML dependencies: liblttng-ust0
```

Comment:
Minor change - to bring the library comments above.

```suggestion
# Install Python dependencies --
# MS SQL related packages: unixodbc msodbcsql17 mssql-tools
# .NET Core dependencies: --no-install-recommends ca-certificates libc6 libgcc1 libgssapi-krb5-2 libicu67 libssl1.1 libstdc++6 zlib1g
# OpenCV dependencies:libglib2.0-0 libsm6 libxext6 libxrender-dev xvfb
# GNU Binutils: binutils
# OpenMP dependencies: libgomp1
# Azure ML dependencies: liblttng-ust0
RUN apt-get update && \
    apt-get install -y wget apt-transport-https curl gnupg locales && \
    echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
    curl https://packages.microsoft.com/config/debian/11/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
    # Needed for libss1.0.0 and in turn MS SQL
    echo 'deb http://security.debian.org/debian-security stable-security main' >> /etc/apt/sources.list && \
    # install necessary locales for MS SQL
    echo 'en_US.UTF-8 UTF-8' > /etc/locale.gen && \
    locale-gen && \
    apt-get update && \
    ACCEPT_EULA=Y apt-get install -y unixodbc msodbcsql17 mssql-tools &&\
    apt-get install -y --no-install-recommends ca-certificates \
    libc6 libgcc1 libgssapi-krb5-2 libicu67 libssl1.1 libstdc++6 zlib1g &&\
    apt-get install -y libglib2.0-0 libsm6 libxext6 libxrender-dev xvfb binutils\
    binutils libgomp1 liblttng-ust0 && \
    rm -rf /var/lib/apt/lists/*
```

Reply:
And if this can be done on other places?

Reply:
Updated. Thanks

Reply:
In my opinion the inline comments are easier to read and remove the possibility of missing an add or remove in the future.  Inline comments make clear which libs are for what and don't come with a risk of adding a library without a defined reason. Doesn't matter either way though just adding my 2c

Reply:
We will discuss internally and update it in future PR. Thanks